### ISSUE TITLE
[Feature] tint based on updated status

### ISSUE THREAD

**Issue by nobless** [HugoHasenbein/redmine_tint_issues#3]
Hello, maybe there is possibility to add support, to tint issues based on updated time, not issue creation time ?, for example if some of issues is updated more than 10 days ago it will be marked as yellow.

Editing colors is settings menu also would be very nice!

Thanks for the plugin!

**Comment by HugoHasenbein** [comment]
Dear Nobles,

you may want to try version 1.2.0. It will support exactly, what you want.

Please give feedback, if that version supports your request.

best regards

Stephan

**Comment by HugoHasenbein** [comment]
Version 1.2.0 does exact this

**Comment by nobless** [comment]
not working,



Showing in the browser:

```
Internal error
An error occurred on the page you were trying to access.
If you continue to experience problems please contact your Redmine administrator for assistance.

If you are the Redmine administrator, check your log files for details about the error.

Back
```


logs:

```

  Rendered plugins/redmine_tint_issues/app/views/hooks/redmine_tint_issues/_tint_issues_css.html.erb (1.3ms)
Completed 500 Internal Server Error in 86ms (ActiveRecord: 14.0ms)

ActionView::Template::Error (undefined method `gsub' for nil:NilClass):
    2:   <%# stylesheet_link_tag "redmine_tint_issues.css", :plugin => "redmine_tint_issues" %>
    3: <style>
    4: /* current */
    5: table.list tbody tr.current:nth-child(odd)         { background-color:<%= Setting['plugin_redmine_tint_issues']['current_issue_age_color'] %>; }
    6: table.list tbody tr.current:nth-child(even)        { background-color:<%= rti_lighten_css_color(  Setting['plugin_redmine_tint_issues']['current_issue_age_color'], 0.05) %>; }
    7: table.list tbody tr.current:nth-child(odd):hover   { background-color:<%= rti_contrast_css_color( Setting['plugin_redmine_tint_issues']['current_issue_age_color']) %>; }
    8: table.list tbody tr.current:nth-child(even):hover  { background-color:<%= rti_contrast_css_color( Setting['plugin_redmine_tint_issues']['current_issue_age_color']) %>; }

plugins/redmine_tint_issues/lib/redmine_tint_issues/patches/application_helper_patch.rb:45:in `rti_lighten_css_color'
plugins/redmine_tint_issues/app/views/hooks/redmine_tint_issues/_tint_issues_css.html.erb:5:in `block in _69a22a61adba02d2dc8048e8e593fcb1'
plugins/redmine_tint_issues/app/views/hooks/redmine_tint_issues/_tint_issues_css.html.erb:1:in `_69a22a61adba02d2dc8048e8e593fcb1'
lib/redmine/hook/view_listener.rb:61:in `block (2 levels) in render_on'
lib/redmine/hook/view_listener.rb:59:in `map'
lib/redmine/hook/view_listener.rb:59:in `block in render_on'
lib/redmine/hook.rb:63:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:63:in `each'
lib/redmine/hook.rb:63:in `block in call_hook'
lib/redmine/hook.rb:60:in `tap'
lib/redmine/hook.rb:60:in `call_hook'
lib/redmine/hook.rb:98:in `call_hook'
app/views/layouts/base.html.erb:16:in `_7a4fe6f7400e717633c07abf7eeaab14'
```

**Comment by HugoHasenbein** [comment]
should now be fixed, version 1.2.1

**Comment by nobless** [comment]
Works now as expected! :) Thanks 👍  


Maybe there is also the possibility to add a feature for filtering which tickets are tinted for example from manager group issues are not tinted if it is in specific group, and for example, maybe there could be a multi-filter exclusion to exclude some groups, as an example in progress or resolved tickets are not tinted.
